README.md

```diff
@@ -54,6 +54,18 @@ It can be generated by running command `make generate-openapi-test`
 Generated spec can be found under 
 `build/rest/openapi/spec`
 
+##### Verifiable Credential Test Suite
+In order to check compatibility of Verifiable Credential implementation in aries-framework-go with 
+[W3C Verifiable Claims Working Group Test Suite](https://github.com/w3c/vc-test-suite), run `make vc-test-suite`.
+The result of Test Suite run is placed under `vc-test-suite/suite/implementations/aries-framework-go-report.json`.
+In order to public the results to the community, run:
```

Comment:
I don't think we need lines 61-67 in the README

Reply:
OK

Reply:
OK